tests/core/test_views_swagger.py

```diff
@@ -37,3 +37,25 @@ def test_extensions(self):
             with mock.patch('pkg_resources.resource_filename', lambda pkg, f: path):
                 response = self.app.get('/swagger.json')
                 self.assertEquals(response.json['swagger'], '3.0')
+
+    def test_default_security_extensions(self):
+        path = tempfile.mktemp(suffix='.yaml')
+        content = {
+            'securityDefinitions': {
+                'fxa': {
+                    'type': 'oauth2',
+                    'authorizationUrl': 'https://oauth-stable.dev.lcip.org',
+                    'flow': 'implicit',
+                    'scopes': [{'kinto': 'Basic scope'}]
+                }
+            }
+        }
```

Comment:
I don't really understand the value of this change. It seems like in this test, the `type` would be something that a client might want to know?